reporter/src/funTest/assets/evaluated-model-reporter-test-deduplicate-expected-output.yml

```diff
@@ -499,10 +499,10 @@ packages:
   declared_licenses:
   - 12
   declared_licenses_processed:
-    spdx_expression: "EPL-1.0 OR MPL-2.0"
+    spdx_expression: "MPL-2.0 OR EPL-1.0"
```

Comment:
Why are SPDX expression terms not not sorted alphabetically anymore?

Reply:
Hm, good question I hadn't notice that it's now in reversed order.
The update was done programmatically. Have to investigate that...